tests/azul_agent.py

```diff
@@ -18,13 +17,13 @@ def __init__(self, deployment):
         azul_adapter = requests.adapters.HTTPAdapter(max_retries=azul_retries)
         self.https_session.mount('https://', azul_adapter)
 
-    def get_files_py_project(self, project_shortname):
+    def get_entity_type_by_project(self, entity, project_shortname):
```

Comment:
```suggestion
    def get_entities_by_project(self, entity_type, project_shortname):
```

This method isn't returning an entity type, it's returning entities of a given type.